src/test/java/org/starchartlabs/flare/plugins/test/plugin/BintrayCredentialsPluginIntegrationTest.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2019 StarChart-Labs@github.com Authors
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+package org.starchartlabs.flare.plugins.test.plugin;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.gradle.testkit.runner.BuildResult;
+import org.gradle.testkit.runner.GradleRunner;
+import org.gradle.testkit.runner.TaskOutcome;
+import org.starchartlabs.flare.plugins.test.IntegrationTestListener;
+import org.starchartlabs.flare.plugins.test.TestGradleProject;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Listeners;
+import org.testng.annotations.Test;
+
+@Listeners(value = { IntegrationTestListener.class })
+public class BintrayCredentialsPluginIntegrationTest {
+
+    private static final Path BUILD_FILE_DIRECTORY = Paths.get("org", "starchartlabs", "flare", "plugins", "test",
+            "bintray", "credentials");
+
+    private static final Path BUILD_FILE = BUILD_FILE_DIRECTORY.resolve("build.gradle");
+
+    private Path projectPath;
+
+    @BeforeClass
+    public void setup() throws Exception {
+        projectPath = TestGradleProject.builder(BUILD_FILE)
+                .build()
+                .getProjectDirectory();
+    }
+
+    // TODO romeara Try to figure out a way to test the environment variable scenario
```

Comment:
Are you intending to do this now or later?

Reply:
Later - this is the same issue as testing the standard managed-credentials environment variable setup

Reply:
Gotcha, ok